app/services/deadline_calculator/calendar_month.rb

```diff
@@ -0,0 +1,39 @@
+# This calculator is based on the rules for ICO
+# - calender month 
+# - if non-business-day, find the closet future working day
+
+module DeadlineCalculator
+    class CalendarMonth
+  
+      attr_reader :kase
+  
+      def initialize(kase)
+        @kase = kase
+      end
+  
+      def escalation_deadline
+        kase.created_at.to_date + kase.correspondence_type.escalation_time_limit.days
```

Comment:
Probably nothing to do with your PR, but wondering why this `to_date` here but not on the other deadline fields 🤔